layers/calendar/config.json

```
@@ -0,0 +1,7 @@
+{
+    "php": [
+        "73",
+        "74",
+        "80",
```

This is not valid json. 

```suggestion
        "80"
```